<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParserStrategyPage</class>
 <widget class="QWidget" name="ParserStrategyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>解析策略管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="text">
        <string>← 返回</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>解析策略管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="strategySelectLayout">
     <item>
      <widget class="QLabel" name="labelSelect">
       <property name="text">
        <string>选择策略:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboStrategies">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNew">
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>454</width>
        <height>362</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>12</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>策略名称:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>输入策略名称</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelSingle">
         <property name="toolTip">
          <string>逗号分隔的关键词列表</string>
         </property>
         <property name="text">
          <string>单选题关键词:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editSingleKeywords">
         <property name="placeholderText">
          <string>例如: 单选题,单选</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelMulti">
         <property name="text">
          <string>多选题关键词:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editMultiKeywords">
         <property name="placeholderText">
          <string>例如: 多选题,多选</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelJudge">
         <property name="text">
          <string>判断题关键词:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editJudgeKeywords">
         <property name="placeholderText">
          <string>例如: 判断题,判断</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelFill">
         <property name="text">
          <string>填空题关键词:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="editFillKeywords">
         <property name="placeholderText">
          <string>例如: 填空题,填空</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelAnswer">
         <property name="toolTip">
          <string>用于识别答案行的关键词</string>
         </property>
         <property name="text">
          <string>答案关键词:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="editAnswerKeywords">
         <property name="placeholderText">
          <string>例如: 正确答案,答案：,答案:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelJudgeTrue">
         <property name="toolTip">
          <string>表示&quot;对&quot;的答案值，逗号分隔</string>
         </property>
         <property name="text">
          <string>判断题&quot;对&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="editJudgeTrue">
         <property name="placeholderText">
          <string>例如: 对,T,true,√,正确</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelJudgeFalse">
         <property name="toolTip">
          <string>表示&quot;错&quot;的答案值，逗号分隔</string>
         </property>
         <property name="text">
          <string>判断题&quot;错&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="editJudgeFalse">
         <property name="placeholderText">
          <string>例如: 错,F,false,×,X,错误</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelGarbage">
         <property name="toolTip">
          <string>需要过滤的垃圾行模式，正则或关键词，逗号分隔</string>
         </property>
         <property name="text">
          <string>过滤规则:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPlainTextEdit" name="editGarbagePatterns">
         <property name="placeholderText">
          <string>每行一个过滤规则，可使用正则表达式</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="toolTip">
        <string>选择文件测试当前策略</string>
       </property>
       <property name="text">
        <string>测试...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupReference">
     <property name="title">
      <string>📖 运行时 C++ 函数参考</string>
     </property>
     <layout class="QVBoxLayout" name="layoutReference">
      <item>
       <widget class="QScrollArea" name="scrollAreaReference">
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollContentReference">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>444</width>
           <height>139</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="layoutReferenceContent">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
